src/main/java/org/eclipse/pass/schema/SchemaFetcher.java

```diff
@@ -40,36 +76,53 @@ private List<SchemaInstance> getRepositorySchemas(String repositoryUri) {
         // Get the schema URIs from the Repository using the getSchemas() method
         // Call resolveSchemaUri() for each schema URI and create a list of
         // returned SchemaInstance objects
-
-        return null;
+        ObjectMapper mapper = new ObjectMapper();
+        Repository r1;
+        List<SchemaInstance> repository_schemas = new ArrayList<SchemaInstance>();
+
+        try {
+            r1 = mapper.readValue(new URL(repositoryUri), Repository.class);
```

Comment:
Ok, here is what I was thinking when I put it together this way. Let me know if you'd still like me to change it though. Since PassClient is an interface, I am using the Repository class which extends this interface. r1 is an instance of a Repository in this case. I am assigning it a value based on the repository URI that I want to create an object for. It's done the same way in line 47 of this class: https://github.com/eclipse-pass/pass-java-client/blob/53d694c6945196d70a401e0bf02fe106f8a96e60/pass-model/src/test/java/org/dataconservancy/pass/model/RepositoryModelTests.java

Reply:
ObjectMapper is more on the side of the internal implementation of how the pass model is mapped to JSON and back. It should generally be avoided for interacting the the PASS model. Those unit tests were checking the implementation and making sure the annotations were correct. 

Instead you should use PassClient.readResource. See https://github.com/eclipse-pass/pass-java-client/blob/main/pass-client-api/src/main/java/org/dataconservancy/pass/client/PassClient.java.